test/core/libp2p/crypto/CMakeLists.txt

```diff
@@ -13,3 +13,25 @@ target_link_libraries(keys_test
         crypto_provider
         Boost::boost
         )
+
+addtest(aes_test
+        aes_test.cpp
+        )
+
+target_link_libraries(aes_test
+        buffer
+        crypto_provider
+        OpenSSL::Crypto
+        Boost::boost
```

Comment:
Most of those libraries are already linked with `crypto_provider`

Reply:
removed redundant libraries links